.circleci/config.yml

```diff
@@ -1,48 +1,60 @@
-version: 2
+version: 2.1
+
+executors:
+  node:
```

Comment:
How about node-latest ?

Reply:
If we would do something with the node version then version should be a param on the executor